hw/ip/dcd/data/dcd.hjson

```diff
@@ -138,13 +144,13 @@
         cname: "DCD",
```

Comment:
nit: there seem to be some non-ASCII apostrophes in this hjson, can you replace them with `'`?

Reply:
thanks for noticing that. removed.